packages/imagelib/imagelib.20160413/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "imagelib"
```

Comment:
Can you remove the `name` and `version` of that file? They are implicit in the directory name already. Thanks!